conda/base/context.py

```diff
@@ -475,7 +475,13 @@ def arch_name(self):
 
     @property
     def conda_private(self):
-        return conda_in_private_env()
+        warnings.warn(
+            "`conda.base.context.context.conda_private` is pending deprecation and will be "
+            "removed in a future release. It's meaningless and any special meaning it may have "
```

Comment:
Because of the wording here, I'm unsure if the history about this is known to everyone.

This was a feature planned for 4.4 which would've helped (and still would) to protect the Conda installation against unintentional breakage. The idea was to make the `base` (back then `root`) environment empty (or populated by the non-`conda` deps of the `anaconda` package) and have `conda` installed into `_conda_`, neatly tucked away from everything else.

This was conceptually/structurally a big change that touched a lot of different bits and pieces, i.e., it was not an easy feat but ultimately/unfortunately resource constraints hindered its completion.
Though by now, saying it doesn't have much meaning anymore is apt, I guess.

See https://github.com/conda/conda/issues/3912 for context.

Reply:
(And right after writing this I saw https://github.com/conda/conda/issues/11342 mentioned https://github.com/conda/conda/issues/3912 :smile: .)